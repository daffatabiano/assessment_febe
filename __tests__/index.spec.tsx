import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import Home from '@/pages/index';

describe('Page', () => {
  it('renders homepage unchanged', () => {
    const { container } = render(<Home />);
    expect(container).toMatchSnapshot();
  });
});
