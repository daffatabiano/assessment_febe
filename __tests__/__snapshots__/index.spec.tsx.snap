// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page renders homepage unchanged 1`] = `
<div>
  <div
    class="text-3xl text-rose-400 text-balance"
  >
     HELLOW RWOEWL!
  </div>
</div>
`;
